### PR TITLE
host-resolve: invalidate ivykis "now" after blocking calls to DNS

### PR DESCRIPTION
**Pull request by bazsi**
Signed-off-by: Balazs Scheidler <bazsi77@gmail.com>

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: modules/afsocket/afinet-dest.c [hunk 1]
```diff
@@ -362,6 +362,7 @@ afinet_dd_setup_addresses(AFSocketDestDriver *s)
   if (!_setup_dest_addr(self))
     return FALSE;
 
+  iv_invalidate_now();
```
Comment (MrAnno) [inline modules/afsocket/afinet-dest.c:365]: I think this is not necessary, as `_setup_dest_addr()` invalidates `now` through `resolve_hostname_to_sockaddr()`.

**Review by MrAnno — APPROVED** [review]
I pushed a small change onto this branch (`#include <iv.h>`) to make everything green.
This fixes #3514.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by gaborznagy — APPROVED** [review]
I agree with @MrAnno's comment that iv_invalide_now() is not needed in afinet-dest.c, otherwise looks good to me.

@bazsi please consider that comment, but it can be merged if you like to keep it this way.

**Comment by MrAnno** [comment]
My comment is not really important, the extra `iv_invalidate_now()` in afinet-dest won't have side-effects.

I'll send a newsfile entry.

**Pull request merged**